pydoctor/sphinx_ext/__init__.py

```diff
@@ -1,3 +1,58 @@
 """
 Public and private extensions for Sphinx.
 """
+import os
+import subprocess
+from pprint import pprint
+
+
+def get_git_reference(main_branch: str = 'master', debug: bool = False, ) -> str:
```

Comment:
```suggestion
def get_git_reference(main_branch: str = 'master', debug: bool = False) -> str:
```
Note that if the RTD code is moved out (see below), we don't need `main_branch` as an argument anymore.

Reply:
Yes. I plan to completely remove the idea of "main_branch" and just link to SHA